Modules/mathmodule.c

```diff
@@ -3335,6 +3398,7 @@ static PyMethodDef math_methods[] = {
     {"asin",            math_asin,      METH_O,         math_asin_doc},
     {"asinh",           math_asinh,     METH_O,         math_asinh_doc},
     {"atan",            math_atan,      METH_O,         math_atan_doc},
+    {"lcm",             math_lcm,       METH_O,         math_lcm_doc},
```

Comment:
Since you're using argument clinic, this should be replaced with `MATH_LCM_METHODDEF` (just like `MATH_GCD_METHODDEF`). The entries in this section are organized alphabetically, so it would go immediately above the existing `MATH_LDEXP_METHODDEF`. You can remove `math_lcm_doc`: the docstring is already present in the argument clinic header.

Reply:
Already noted in private conversation, but worth adding here for the public record: you'll need to regenerate the autogenerated mathmodule.c.h file. You should be able to do that from a shell prompt (`$`) with something like:
```bash
$ python Tools/clinic/clinic.py Modules/mathmodule.c
```
That will then make `MATH_LDEXP_METHODDEF` available for use in this module.

Reply:
Hmm. I'm out date. `make clinic` appears to be an easier way to do this.